src/GazeboThruster.cpp

```diff
@@ -140,6 +148,16 @@ void GazeboThruster::readInput(ThrustersMSG const& thrustersMSG)
     }
 }
 
+void GazeboThruster::readCompensatedMass(CompMassMSG const& compMassMSG)
+{
+    gazebo_underwater::Matrix6 matrix(compMassMSG->matrix());
+    gazebo::math::Vector3 cog(gazebo::msgs::ConvertIgn(compMassMSG->cog()));
+    if(!compensated_effort)
```

Comment:
I could not find a way to publish or subscribe one time only. 
The flag was a quick solution for not update the value every step.